docs/presets/jetpack.md

```diff
@@ -0,0 +1,25 @@
+[Return to Presets](./README.md#jetpack)
+
+# Jetpack Preset
+
+This is a pseudo minimalist preset inspired by the [geometry](https://github.com/geometry-zsh/geometry) and [spaceship](https://github.com/spaceship-prompt/spaceship-prompt) prompts.
+
+> Jetpack uses the terminal's color theme.
+
+![Screenshot of Jetpack preset](/presets/img/jetpack.png)
+
+### Prerequisite
+
+- [Jetbrains Mono](https://www.jetbrains.com/lp/mono/) installed and enabled in the terminal.
+
+### Manual Configuration
+
+[Click to download TOML](/presets/toml/jetpack.toml)
+
+### Configuration
+
+```sh
+starship preset jetpack-o ~/.config/starship.toml
+```
+
```

Comment:
All other presets offer a line to download the _TOML_ if one does prefer. I would recommend adding it—see [Nerd Font Symbols Preset](https://starship.rs/presets/nerd-font.html).

Reply:
@mickimnet Did I use the incorrect format for downloads in [line 19](https://github.com/Indyandie/starship/blob/bc255473b4371bbbf3e23b5fbd689403a1c9f175/docs/presets/jetpack.md?plain=1#L18)?

Reply:
I'll check later today or tomorrow.